packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx

```diff
@@ -274,13 +279,32 @@ export class TreeViewWidget extends TreeViewWelcomeWidget {
             classes.push(TREE_NODE_SEGMENT_GROW_CLASS);
         }
         const className = classes.join(' ');
-        const title = node.tooltip ||
-            (node.resourceUri && this.labelProvider.getLongName(new CoreURI(node.resourceUri)))
-            || this.toNodeName(node);
-        const attrs = this.decorateCaption(node, {
-            className, id: node.id,
-            title
-        });
+
+        let attrs: {} = {
```

Comment:
If you export the `TooltipAttributes` interface I suggested above, this could be typed like this:

```typescript
let attrs: React.HTMLAttributes<HTMLElement> & Partial<TooltipAttributes> = ...
```